### PR TITLE
Multiple Experiments Management UX

### PR DESCRIPTION
**Pull request by Lijiaoa**
related issue #2779 #3100 https://github.com/microsoft/nni/issues/3055

**### now, experiment still click `Name` column to other webui, also with `<a>` style.**

- [x] intermediate graph(compare.tsx) point hover show 'N/A' #3118
- [x] add error status
- [x] adjust column width on screen 1490px (1490px is max width now)
- [x] test

### CODE REVIEW

File: ts/webui/src/components/managementExp/experimentConst.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { IDatePickerStrings } from '@fluentui/react';
+const DayPickerStrings: IDatePickerStrings = {
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/experimentConst.ts:2]: Is this necessary? The doc says `strings` is optional.

File: ts/webui/src/static/style/common/common.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.ellipsis {
```
Comment (liuzhe-lz) [inline ts/webui/src/static/style/common/common.scss:1]: The path looks a little bit strange. Why it's not `style/common.scss`?

File: ts/webui/src/components/managementExp/Experiment.tsx [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+            name: 'ID',
+            key: 'id',
+            fieldName: 'id',
+            minWidth: MINSCREENCOLUMNWIDHT,
+            maxWidth: MAXSCREENCOLUMNWIDHT,
+            isResizable: true,
+            className: 'tableHead leftTitle',
+            data: 'string',
+            onColumnClick: this.onColumnClick,
+            onRender: (item: any): React.ReactNode => <div className='succeed-padding'>{item.id}</div>
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/Experiment.tsx:181]: Suggest to make "ID" and "Status" clickable.
And personally suggest to remove underscore from "Name".

File: ts/webui/src/components/managementExp/NameColumn.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    render(): React.ReactNode {
+        const { port, expName } = this.props;
+        const hostname = window.location.hostname;
+        const protocol = window.location.protocol;
+        const webuiPortal = `${protocol}//${hostname}:${port}/oview`;
+        return (
+            <div className='succeed-padding ellipsis'>
+                <a href={webuiPortal} className='link' target='_blank' rel='noopener noreferrer'>
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/NameColumn.tsx:20]: Stopped experiments should not be clickable.

And @J-shang , stopped experiments should not have port in `.experiment`.

File: ts/webui/src/components/managementExp/NameColumn.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+class NameColumn extends React.Component<NameColumnProps, {}> {
+    constructor(props: NameColumnProps) {
+        super(props);
+    }
+
+    render(): React.ReactNode {
+        const { port, expName } = this.props;
+        const hostname = window.location.hostname;
+        const protocol = window.location.protocol;
+        const webuiPortal = `${protocol}//${hostname}:${port}/oview`;
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/NameColumn.tsx:17]: Stopped experiments should not be clickable.

And @J-shang , stopped experiments should not have port in `.experiment`.

File: ts/webui/src/components/managementExp/FilterBtns.tsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+            <React.Fragment>
+                <Dropdown
+                    label='Status'
+                    selectedKey={selectedStatus}
+                    onChange={selectStatus.bind(this)}
+                    placeholder='Select an option'
+                    options={fillOptions(EXPERIMENTSTATUS)}
+                    className='filter-condition-status'
+                />
+                <Dropdown
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/FilterBtns.tsx:47]: When there is only one platform, this dropdown does not work.

File: ts/webui/src/components/managementExp/Experiment.tsx [hunk 7]
```diff
@@ -0,0 +1,433 @@
...
+    selectedPlatform: string;
+    selectedStartDate?: Date;
+    selectedEndDate?: Date;
+    sortInfo: SortInfo;
+    source: AllExperimentList[];
+    originExperimentList: AllExperimentList[];
+    searchSource: AllExperimentList[];
+}
+
+class Experiment extends React.Component<{}, ExpListState> {
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/Experiment.tsx:33]: Suggest to rename to `ExperimentManager` or something similar. Current name looks like "one experiment"..

File: ts/webui/src/components/managementExp/Experiment.tsx [hunk 8]
```diff
@@ -0,0 +1,433 @@
...
+                                    selectedEndDate={selectedEndDate!}
+                                    selectStatus={this.selectStatus.bind(this)}
+                                    selectPlatform={this.selectPlatform.bind(this)}
+                                    getSelectedData={this.getSelectedData.bind(this)}
+                                    setSearchSource={this.setSearchSource.bind(this)}
+                                />
+                            </Stack>
+                            <DetailsList
+                                columns={this.columns}
+                                items={source}
```
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/Experiment.tsx:123]: Is this supposed to be auto-refreshing?

File: ts/webui/src/components/managementExp/Experiment.tsx (hunk 7 above)
Comment (liuzhe-lz) [inline ts/webui/src/components/managementExp/Experiment.tsx:33]: Suggest to rename to `ExperimentManager` or something similar. Current name looks like "one experiment".

File: ts/webui/src/components/managementExp/NameColumn.tsx (hunk 5 above)
Comment (J-shang) [reply ts/webui/src/components/managementExp/NameColumn.tsx:17]: OK, will remove it.

File: ts/webui/src/components/managementExp/Experiment.tsx (hunk 7 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/Experiment.tsx:33]: ok

File: ts/webui/src/components/managementExp/Experiment.tsx (hunk 8 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/Experiment.tsx:123]: in this version, it has not auto-refresh

File: ts/webui/src/components/managementExp/NameColumn.tsx (hunk 4 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/NameColumn.tsx:20]: by design, i will discuss with Xingyu

File: ts/webui/src/static/style/common/common.scss (hunk 2 above)
Comment (Lijiaoa) [reply ts/webui/src/static/style/common/common.scss:1]: thanks~ it's easy to refine.

File: ts/webui/src/components/managementExp/FilterBtns.tsx (hunk 6 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/FilterBtns.tsx:47]: Thanks for finding that! I'll fix this issue.

**Review by colorjam — APPROVED** [review]

File: ts/webui/src/components/managementExp/experimentConst.ts (hunk 1 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/experimentConst.ts]: Thanks for your suggestion. It's optional, i had delete related code.

File: ts/webui/src/components/managementExp/Experiment.tsx (hunk 3 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/Experiment.tsx]: discuss with xingyu, and now `ID`  column is clickable.

File: ts/webui/src/components/managementExp/NameColumn.tsx (hunk 4 above)
Comment (Lijiaoa) [reply ts/webui/src/components/managementExp/NameColumn.tsx]: now, when cursor hover on the trial id, font change `blue with underline`

**Review by liuzhe-lz — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Pull request merged**